inc/vcf/file_structure.hpp

```diff
@@ -171,15 +171,43 @@ namespace ebi
          * @throw std::invalid_argument
          */
         void check_chromosome() const;
-        
+
         /**
-         * Checks that IDs are alphanumeric
+         * Checks that chromosome does not contain any colons
+         * 
+         * @throw std::invalid_argument
```

Comment:
So, should I remove this particular line `@throw std::invalid_argument` from the comments for all the methods? In that case, would it be okay if I also changed the comment style from multi line comments `/*   */` to single line comments `//   ` as there would only be one line for most of the comments sections.